aptos-move/aptos-vm/src/zkid_validation.rs

```diff
@@ -132,6 +141,10 @@ pub fn validate_zkid_authenticators(
         match &zkid_sig.sig {
             ZkpOrOpenIdSig::Groth16Zkp(proof) => match jwk {
                 JWK::RSA(rsa_jwk) => {
+                    if proof.exp_horizon_secs > config.max_exp_horizon_secs {
```

Comment:
can you add a todo for me to write a smoke test for this later

Reply:
I think there is one in the smoke tests